services/core-web/src/components/mine/MineDashboard.js

```diff
@@ -305,8 +304,7 @@ export class MineDashboard extends Component {
             </div>
           )}
         </AuthorizationWrapper>
-        {/* this is an external link to mineSpace, only available to users in production as its a prod URL, not using the AuthWrapper since the admin role overrides the wrapper. */}
-        {detectProdEnvironment() && (
+        {/* this is an external link to mineSpace, not using the AuthWrapper since the admin role overrides the wrapper. */}        
```

Comment:
This is a testing consideration for Minespace! The login process there has a case for handling redirects from Core- this will send the user to the mine on MS with the url param redirectFromCore=true
Making it possible to test outside of prod